docs/DeveloperGuide.md

```diff
@@ -8,22 +8,107 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### _Designs_
+### 1. Calories Management Mode
+![Commands for managing calories](./diagrams/CaloriesManagement.png)
+   * when `caloriesManagement` is executed in Parser, different actions will be carried out based on the commands.
+   * if `eat`, store the name and calories of the input food
+   * if `view`, display all the foods consumed
+
+### 1. View all groceries added
+   * First create a method in "Grocery" class that prints the grocery in a preferred format.\
+     e.g., NAME, AMOUNT, EXPIRATION, PRICE.
```

Comment:
Can you provide a sample of the formats for all those inputs? (I can see you have already specified the format of expiration, but maybe a sample for all is more intuitive for other developers)